SwrveSDKCommon/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '23.0.2'
```

Comment:
Yeah, I had that in, then when I checked the other guys, those settings weren't in any of the public guys (like SwrveSDK proper).  The issue there would be that that properties file lives in private area, so if someone clones this public dude, that file wouldn't be there.  So that means there needs to be one in the public area...

In general, believe me, I debated this issue in my own head before committing and just thought it'd be better to get this in, and this PR done, then discuss cleaning that side up.  Baby steps!